### PR TITLE
Remove emails from airtable

### PR DESCRIPTION
**Pull request by weotch**
Removes posting (and fetching) emails from Airtable and sends it, instead, as part of the `User Logged In` Amplitude event payload.

### CODE REVIEW

File: flappy/main.js [hunk 1]
```diff
@@ -378,7 +378,6 @@ function logHighscore(score, duration, vehicle, stage) {
 
     const table = EASY_MODE ? base('Data Explorer Scores') : base('Data Explorer Scores (Hard)');
     table.create({
-        'name': userName,
```
Comment (hacos) [inline flappy/main.js:381]: I assume this is intentional to remove userName from the table?

**Review by hacos — APPROVED** [review]
lgtm from the perspective that we remove email from airtable

File: flappy/main.js (hunk 1 above)
Comment (weotch) [reply flappy/main.js:381]: Yeah, since `userName` is an email

**Review by Sierangho — APPROVED** [review]
lgtm - I assume stuff still works

**Pull request merged**